scripts/e2e/setup_fixtures

```diff
@@ -1,13 +1,44 @@
 #!/bin/bash -eu
-
 # we assume that the admin user already exists
 
-echo && echo "[$0] creating the PARENT_PLACE..."
+delete_doc() {
+	echo "[$0] Deleting doc $1"
+	local docUrl="$COUCH_URL/$1"
+	local rev="$(curl $docUrl | jq --raw-output ._rev)"
+	if [[ "$rev" != "null" ]]; then
+		curl -X DELETE "$docUrl?rev=$rev"
+	fi
+}
+
+delete_user() {
+	echo "[$0] Deleting user $1"
+	local username="$1"
+	curl -X DELETE "$API_URL/api/v1/users/$1"
+}
+
+echo "[$0] Setting up fixtures for api e2e test. Cleaning up old test data..."
+
+delete_user "bob"
+delete_user "clare"
+delete_user "chw-boss"
+delete_user "chw"
+
+delete_doc "PARENT_PLACE"
+delete_doc "fixture:bobville"
+delete_doc "fixture:clareville"
+delete_doc "fixture:chw-bossville"
+delete_doc "fixture:chwville"
+delete_doc "fixture:user:chw"
+delete_doc "fixture:user:chw-boss"
+
+echo "[$0] Finished cleanup. Creating new test data..."
+
+echo "[$0] creating the PARENT_PLACE..."
 curl -X PUT \
 	--data '{ "_id":"PARENT_PLACE", "type":"district_hospital", "name":"Big Parent Hospital" }' \
 	"$COUCH_URL/PARENT_PLACE"
 
-echo && echo "[$0] creating user: bob..."
```

Comment:
Whatever they were for, I can't see a difference either locally or in travis... I can research more if you think it's important, but that's enough for my curiosity.